src/sentry/monitors/processing_errors/manager.py

```diff
@@ -178,6 +178,13 @@ def handle_processing_errors(item: CheckinItem, error: ProcessingErrorsException
                 "sdk_platform": item.message["sdk"],
             },
         )
+        analytics.record(
```

Comment:
how much latency will this add to the system I wonder

Reply:
yea thats a fair point. I guess the main motivation for doing something like this is so that as we roll out the feature to a number of orgs we can check in on them and make sure the errors they are receiving make sense. Would a better pattern to take this out of the critical ingest code path be to use signals?

Reply:
nvm i'm not sure if these signals work async the way i'm thinking about

Reply:
follow up here, after talking with the team, it seems like adding one call to analytics shouldn't be a huge problem